service/controller.go

```diff
@@ -361,9 +374,47 @@ func (s *service) CreateVolume(
 			return nil, status.Errorf(codes.Unknown, "Create Volume %s failed with error: %v", volName, err)
 		}
 
+		// set quota limits, if specified in NFS storage class
+		isQuotaEnabled := s.opts.IsQuotaEnabled
+		if isQuotaEnabled {
+			// get filesystem (NFS volume), newly created
+			fs, err := system.GetFileSystemByIDName(fsResp.ID, "")
+			if err != nil {
+				Log.Debugf("Find Volume response error: %v", err)
+				return nil, status.Errorf(codes.Unknown, "Find Volume response error: %v", err)
```

Comment:
I think here codes.notfound or codes.internal will be a great fit rather than codes.unknow thoughts?

Reply:
Hi @VamsiSiddu-7, I referred to the existing codes used for such error scenarios, hence went ahead to maintain the same responses.
https://github.com/dell/csi-powerflex/blob/main/service/controller.go#L361 
If we plan to change it here, then it should be reflected to other instances as well to maintain the consistency.

Reply:
okay let's keep it the same codes.Unknown we will have consistency